tests/Coduo/PHPMatcher/MatcherTest.php

```diff
@@ -253,7 +253,9 @@ public static function expanderExamples()
             array(100, "@integer@.lowerThan(101).greaterThan(10)", true),
             array("", "@string@.notEmpty()", false),
             array("lorem ipsum", "@string@.notEmpty()", true),
+            array("", "@string@.isEmpty()", true),
             array(array("foo", "bar"), "@array@.inArray(\"bar\")", true),
+            array(array(), "@array@.isEmpty()", true),
```

Comment:
could you please also add test that proves that for non empty valute this expander returns false?